src/eva/transforms/channel_stats.py

```diff
@@ -0,0 +1,67 @@
+# (C) Copyright 2021-2022 NOAA/NWS/EMC
+#
+# (C) Copyright 2021-2022 United States Government as represented by the Administrator of the
+# National Aeronautics and Space Administration. All Rights Reserved.
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+
+# --------------------------------------------------------------------------------------------------
+
+
+from math import sqrt
+import re
+from statistics import mean
+
+from eva.utilities.config import get
+from eva.utilities.logger import Logger
+from eva.transforms.transform_utils import parse_for_dict, split_collectiongroupvariable
+from eva.transforms.transform_utils import replace_cgv
+from eva.utilities.utils import remove_list_duplicates
+from eva.utilities.utils import remove_empty_from_list_of_strings
+
+
+# --------------------------------------------------------------------------------------------------
+
+
+def channel_stats(config, data_collections):
+
+    # Create a logger
+    logger = Logger('ChannelStatsTransform')
+
+    # Define default stat functions to calculate
+    if 'statistic list' in config:
+        stat_functions = get(config, logger, 'statistic list')
+        print('stat_functions=',stat_functions)
```

Comment:
Use the logger instead?

Reply:
@CoryMartin-NOAA That print statement was left from debugging.   I have removed it.